app-text/poppler/poppler-0.50.0.recipe

```diff
@@ -0,0 +1,238 @@
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	poppler$secondaryArchSuffix = $portVersion
+	lib:libpoppler$secondaryArchSuffix = 65.0.0
+	lib:libpoppler_cpp$secondaryArchSuffix = 0.3.0
+	cmd:pdfdetach$commandSuffix
+	cmd:pdfimages$commandSuffix
+	cmd:pdfseparate$commandSuffix
+	cmd:pdftohtml$commandSuffix
+	cmd:pdftops$commandSuffix
+	cmd:pdfunite$commandSuffix
+	cmd:pdffonts$commandSuffix
+	cmd:pdfinfo$commandSuffix
+	cmd:pdftocairo$commandSuffix
+	cmd:pdftoppm$commandSuffix
+	cmd:pdftotext$commandSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	poppler_data
+	lib:libcairo$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblcms2$secondaryArchSuffix
+	#lib:libnss3$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel
+	devel:libcairo$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblcms2$secondaryArchSuffix
+	#devel:libnss3$secondaryArchSuffix
+	devel:libopenjp2$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gawk
+	cmd:grep
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+BUILD()
+{
+	export CFLAGS="-D_BSD_SOURCE"
+	export CXXFLAGS="-D_BSD_SOURCE"
+	export LDFLAGS="-lnetwork -lbsd"
+
+	libtoolize --force --copy --install
+	aclocal -I m4
+	autoconf
+	automake --add-missing
+	runConfigure --omit-dirs "binDir sbinDir docDir" ./configure \
+		--bindir=$commandBinDir \
+		--sbindir=$commandBinDir \
+		--docdir=$docDir \
+		--enable-xpdf-headers \
+		--enable-build-type=release \
+		--enable-libcurl \
+		--enable-cmyk
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -rf $libDir/*.la
+
+	# defining docDir doesn't helps too much
+	popplerdocdir="$dataDir/gtk-doc/html/poppler"
+	mkdir -p $docDir
+	cp -R $popplerdocdir/* $docDir/
+	rm -rf $dataDir
+
+	prepareInstalledDevelLibs libpoppler \
+		libpoppler-cpp \
+		libpoppler-glib \
+		libpoppler-qt5
+
+	fixPkgconfig
+
+# ============= GLIB =============
```

Comment:
Please remove all =, they can be mistaken when merging. The comment should start aligned vertically.